copter/source/docs/thrust_loss_yaw_imbalance.rst

```diff
@@ -0,0 +1,47 @@
+.. _thrust_loss_yaw_imbalance:
+
+======================================
+Thrust loss and yaw imbalance warnings
+======================================
+
+If you see thrust loss or yaw imbalance warnings this page outlines some checks and modifications that should be done to resolve the issue. In the majority of cases these warnings are the result in incorrect hardware selection or setup.
+
+These warnings are designed to detect hardware failures in the propulsion system, if they begin to trip on a vehicle that has been flying without warnings the propulsion system should be checked.
+Warning are more likely to happen with larger payloads and in strong/gusty winds.
+
+Potential thrust Loss
+=====================
+
+If potential thrust loss messages are seen on the GCS or in the data-flash logs investigation should be done to find the cause and remedy. The warning will give a motor number, for example:
+
+```Potential Thrust Loss (3)```
+
+These warnings are the result of a motor or motors saturating at 100% throttle, because of this saturation ArduCopter can no longer achieve the requested roll, pitch, yaw and throttle output.
```

Comment:
```suggestion
These warnings are the result of a motor or motors saturating at 100% throttle,. Because of this saturation ArduCopter can no longer achieve the requested roll, pitch, yaw and throttle output.
```